std/experimental/allocator/package.d

```diff
@@ -560,6 +560,179 @@ unittest
     test(theAllocator);
 }
 
+// the class repository used in factory()
+package TypeInfo_Class[string] _factoryClasses;
+
+/**
+Registers a class to be available in the $(LREF factory) function.
```

Comment:
It's an error, i've forget to test with -vtls, actually I'll make it `__gshared`, it has to be global.